radareorg/cutter
# Make grouped dock dragging optional.

**Detailed description**

On one hand grouped dragging is required on the other hand Qt builtin implementation doesn't feel right. Make grouped dock dragging optional until better alternative is found. There is already task for evaluating such libraries #1265


**Test plan (required)**

* create tab group with at least 2 tabs
* try dragging by tittle bar, observe that both tabs get dragged
* disable grouped dragging and try dragging by tittle bar, observe that only one tab gets dragged
* restart cutter, observe that grouped dragging is still disabled in menu
* verify that dragging by title bar drags single tab
* enable grouped dragging,
* try dragging by tittle bar, observe that both tabs get dragged
* restart cutter, observe that grouped dragging is still enabled
* try dragging by tittle bar, observe that both tabs get dragged
